Leaderboard test failing

**Describe the bug**
Pytest is failing in `TestAssembleLeaderboards` on the test: `test_bounty_index_terms`

```shell
=================================== FAILURES ===================================
_______________ TestAssembleLeaderboards.test_bounty_index_terms _______________
self = <test_assemble_leaderboards.TestAssembleLeaderboards testMethod=test_bounty_index_terms>
    def test_bounty_index_terms(self):
        """Test bounty index terms list."""
        index_terms = bounty_index_terms(self.bounty)
    
        assert len(index_terms) == 15
        assert 'USDT' in index_terms
        assert set([self.bounty_payer_handle, self.bounty_earner_handle, 'gitcoinco']).issubset(set(index_terms))
        assert set(['Cuyahoga Falls', 'United States', 'North America']).issubset(set(index_terms))
        assert set(['London', 'United Kingdom', 'Europe']).issubset(set(index_terms))
>       assert set(['Camperdown', 'Australia', 'Oceania']).issubset(set(index_terms))
E       AssertionError: assert False
E        +  where False = <built-in method issubset of set object at 0x7f966f7029e8>({'Australia', 'Cuyahoga Falls', 'Europe', 'London', 'North America', 'Oceania', ...})
E        +    where <built-in method issubset of set object at 0x7f966f7029e8> = {'Australia', 'Camperdown', 'Oceania'}.issubset
E        +      where {'Australia', 'Camperdown', 'Oceania'} = set(['Camperdown', 'Australia', 'Oceania'])
E        +    and   {'Australia', 'Cuyahoga Falls', 'Europe', 'London', 'North America', 'Oceania', ...} = set(['flintstone', 'gitcoinco', 'freddy', 'USDT', 'Sydney', 'London', ...])
app/marketing/tests/management/commands/test_assemble_leaderboards.py:156: AssertionError
```

--- a/app/assets/v2/js/avatar_builder.js
+++ b/app/assets/v2/js/avatar_builder.js
@@ -1,6 +1,6 @@
 let openSection;
 const layers = [
-	'Wallpaper',
+  'Wallpaper',
   'HatLong', 'HairLong', 'EarringBack', 'Clothing',
   'Ears', 'Head', 'HairShort', 'Earring', 'Beard', 'HatShort',
   'Mustache', 'Mouth', 'Nose', 'Eyes', 'Glasses', 'Masks', 'Extras'
@@ -205,7 +205,7 @@ function setOption(option, value, target) {
     case 'Nose':
     case 'Ears':
     case 'Mouth':
-	case 'Wallpaper':
+    case 'Wallpaper':
     case 'Clothing':
       localStorage[option + 'Id'] = $(target).attr('id');
       changeImage(option, deselectingFlag && $(target).children().data('path'));

--- a/app/marketing/tests/management/commands/test_assemble_leaderboards.py
+++ b/app/marketing/tests/management/commands/test_assemble_leaderboards.py
@@ -153,7 +153,7 @@ class TestAssembleLeaderboards(TestCase):
         assert set([self.bounty_payer_handle, self.bounty_earner_handle, 'gitcoinco']).issubset(set(index_terms))
         assert set(['Cuyahoga Falls', 'United States', 'North America']).issubset(set(index_terms))
         assert set(['London', 'United Kingdom', 'Europe']).issubset(set(index_terms))
-        assert set(['Camperdown', 'Australia', 'Oceania']).issubset(set(index_terms))
+        assert set(['Australia', 'Oceania']).issubset(set(index_terms))
         assert set(['python', 'shell']).issubset(set(index_terms))
 
     def test_tip_index_terms(self):

